wp-content/themes/haruoshi_origin/contact.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * This is contact.php
+ * 
+ * Template Name: contact
+ *
+ * @package haruoshi-origin
+ */
+
+get_header(); ?>
+
+<article class="container">
```

Comment:
```suggestion
<article class="container contact">
```
cssの影響範囲狭めるために `contact` ってつけてたほうが良さそう

Reply:
やったが具体的にはどういう良いことがある？
https://github.com/oooshima/origin_wordpress/pull/37/commits/68052479b7e64ccabc963571daaa60d48cc09d6f

Reply:
今のままだとなんの意味もない
wp-content/themes/haruoshi_origin/src/scss/page-contact.scss の3行目に
```
.contact {
```
を追加してあげれば影響範囲狭められる

Reply:
done https://github.com/oooshima/origin_wordpress/pull/37/commits/854b958a56374859ffcf6ead86c0f6b02c61c523